examples/gcal_oo_or.ty

```diff
@@ -90,7 +91,16 @@ p.add(
         Learning rate for the lateral excitatory projection to V1."""),
 
     inh_lr=param.Number(default=0.3,bounds=(0.0,None),doc="""
-        Learning rate for the lateral inhibitory projection to V1."""))
+        Learning rate for the lateral inhibitory projection to V1."""),
+
+    gpu=param.Boolean(default=False,bounds=(0,1),doc="""
+        Sets whether the simulation should be run on a GPU"""))
+
+
+if p.gpu:
+    from topo.gpu.projection import GPUSparseCFProjection
+    from topo.gpu.sheet import compute_sparse_gpu_joint_norm_totals, GPUSettlingCFSheet
+
```

Comment:
I'm slightly confused here -- shouldn't gpu=True be an option on examples/sparse_gcal_oo_or.ty, not examples/gcal_oo_or.ty?  I.e., isn't GPU execution only supported for sparse projections?   

I'm also not sure what the status of this file is meant to be, compared to the submodel-based file in topo/tests/gcal_sparse_gpu.ty.  Can the changes to this file and examples/sparse_gcal_oo_or.ty be abandoned and topo/tests/gcal_sparse_gpu.ty be moved into here, supporting either GPU or non GPU sparse projections?